stylesheets/components/pagination/_paginator.styl

```diff
@@ -79,3 +79,10 @@
 
         @media $media-medium
           display: inline
+
+@media (max-width: 767px) {
```

Comment:
You should be able to use `$media-medium-max` here.

Also, can you match the Stylus style? (no semicolons or braces)